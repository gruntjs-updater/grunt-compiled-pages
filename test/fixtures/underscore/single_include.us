<!DOCTYPE html>
<html>
  <head>
    <title>Single Include</title>

    <link rel="stylesheet" type="text/css" href="<%= css %>" media="all" />
  </head>
  <body>
    <%= include('test/fixtures/underscore/hello_world.us') %>
    <script type="text/javascript" src="<%= js %>"></script>
  </body>
</html>