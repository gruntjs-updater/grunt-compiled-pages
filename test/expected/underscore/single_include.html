<!DOCTYPE html>
<html>
  <head>
    <title>Single Include</title>

    <link rel="stylesheet" type="text/css" href="app/styles.css" media="all" />
  </head>
  <body>
    <strong>Hello World!</strong>
    <script type="text/javascript" src="app/scripts.js"></script>
  </body>
</html>